pytensor/tensor/basic.py

```diff
@@ -1671,22 +1667,14 @@ def do_constant_folding(self, fgraph, node):
                 idx == 0
                 and isinstance(
                     client.op,
-                    (
-                        # Ops that will work inplace on the Alloc. So if they
```

Comment:
I don't think it should remove thsi comment.

Reply:
Indeed, incredible catch! I saw a few more like that, I will take a closer look.

Reply:
Ok I got all the comments back, checked with `git diff main | rg '^[+-].*#'`